src/dndbeyond/base/monster.js

```diff
@@ -458,13 +458,26 @@ class Monster extends CharacterBase {
     }
 
     parseAttackInfo(description) {
-        const m = description.match(/(Melee|Ranged)(?: Weapon| Spell)? Attack:.*?(\+[0-9]+) to hit.*?, (?:reach |ranged? |Gibletish )?(.*?)(?:,.*?)?\./i)
+        // changed for 2024 info descriptions
+        const m = description.match(/(Melee|Ranged|Spell)(?: Weapon| Spell| Attack) ?(?: Attack|Roll):.*?(\+[0-9]+)(?: to hit.*?|\s?,) (?:reach |ranged? |Gibletish )?(.*?)(?:,.*?)?\./i);
```

Comment:
Wondering if instead of `/(Melee|Ranged|Spell)(?: Weapon| Spell| Attack) ?(?: Attack|Roll):` we used `/(Melee|Ranged|Spell)(?: Weapon Attack| Spell Attack| Attack Roll) ?:`, if it'd be better.
Extremely minor and not an actual issue.